client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -142,31 +183,72 @@ function AddTestToQueueWithConfirmation({
     };
 
     const renderErrorDialog = () => {
+        let title = 'Conflicting Report Found';
+        let content =
+            'The report could not be created because an existing ' +
+            'report was found on the reports page with the same AT, ' +
+            'browser and test plan version. Would you like to return ' +
+            'the existing report back to the test queue?';
+        let actions = [
+            {
+                label: 'Proceed',
+                onClick: async () => {
+                    setErrorMessage(false);
+                    if (hasAutomationSupport) {
+                        setShowConfirmation(true);
+                    } else {
+                        await addTestToQueue();
+                    }
+                }
+            }
+        ];
+
+        if (oldReportToCopyResultsFrom) {
+            title = 'Older Results Data Found';
+            content =
+                'Older results with the same AT, browser and test plan ' +
+                'were found for the report being created. Would you like ' +
+                'to copy the older results into the report or create a ' +
+                'completely new report?';
+            actions = [
+                {
+                    label: 'Create new report',
```

Comment:
How about "Create empty report"?

Reply:
Sounds good, done in https://github.com/w3c/aria-at-app/pull/978/commits/bffd2247824ed3d639b6576f8aad11085d539030